public/content/en/gems.md

```diff
@@ -79,12 +79,18 @@ void main()
         writeln("  <head>");
         scope(exit) writeln("  </head>");
         "    <title>%s</title>".writefln("Hello");
-    }
+    } // the scope(exit) on the previous line is executed here
```

Comment:
Could you please split up the comment in 2 lines? The column width for source code should be max 48 characters so that it still looks good on movile (and doesn't wrap)